packages/react/src/reactrouterv4v5/global-flags.ts

```diff
@@ -0,0 +1,5 @@
+import type { Client } from '@sentry/types';
+
+export const V4_SETUP_CLIENTS = new WeakMap<Client, boolean>();
+
+export const V5_SETUP_CLIENTS = new WeakMap<Client, boolean>();
```

Comment:
These weak maps are used to validate if the integration should work with `withSentryRouting` helper.